### ISSUE TITLE
Toroko doesn't get captured in a bubble

### ISSUE THREAD

**Issue by sudobash1** [doukutsu-rs/doukutsu-rs#238]
I just started my first Cave Story playthrough since it first came out, and I thought that I'd try doukutsu-rs. I got to the scene where Toroko gets captured by Misery, except she doesn't:

![toroko_is_still_here](https://github.com/doukutsu-rs/doukutsu-rs/assets/1923776/f568fa74-ab12-4638-8bce-d7afaae2a70f)

The bubble goes the wrong way, and Toroko remains in the scene. In fact, I can go up to her and talk to her again, at which point the boss sequence initiates again (although glitched).

This is with the latest build on Linux.

**Comment by sudobash1** [comment]
OK, I did a little looking at history. I suspected this must have been a recent regression because it couldn't have been unnoticed for long. Turns out it is very recent (only 3 hours at time of writing).

If I download a build from before the latest commit (c2ad3dd6430ec0b23ba749fc3f9a15d692e9c215) then the problem goes away.

I haven't built from source yet, so I'm not able to start debugging this. But for now, I am happy to be able to continue playing with the previous build.
